### ISSUE TITLE
Window widgets on prefs dialog not working on macOS on 3.0rc

### ISSUE THREAD

**Issue by virtualritz** [darktable-org/darktable#3319]
See title.
Close, minimize and maximize widget do not work.
This dialog probably should only have a close widget anyway.

The only way to close this dialog is the `close` button at the bottom.

**Comment by parafin** [comment]
Yeah, I noticed it too. I wonder how it works on Linux.

**Comment by junkyardsparkle** [comment]
Works normally for me on Linux.

**Comment by parafin** [comment]
It looks like a bug in GTK. Some other dialog windows show the same behaviour. Seems that clicks get passed to the main window instead of executing action on the dialog window. If you click on the lower part of close/minimize/maximize widgets they actually work. Sometimes it helps to click inside the dialog window, so that it gets focus, then whole widgets begin to be active (doesn't work well for preferences, but works with history stack copy and others). There are other issues with how mouse click events propagate in Quartz GTK, so it looks like a more general problem.
To summarize: won't fix, upstream GTK bug.
